scripts/qa_crowbarsetup.sh

```diff
@@ -1248,7 +1249,7 @@ function onadmin_add_cloud_repo()
         echo "$clouddescription"
         echo
     fi
-    ) >> /etc/motd
+    ) > /etc/motd
```

Comment:
I don't think so. It prints only the info about Cloud media (like `download.nue.suse.com/ibs/Devel:/Cloud:/7:/Staging/images/iso/SUSE-OPENSTACK-CLOUD-7-x86_64-Media1.iso`), which is same for both cases when this function is called